pkg/es/config/config.go

```diff
@@ -366,3 +376,23 @@ func loadToken(path string) (string, error) {
 	}
 	return strings.TrimRight(string(b), "\r\n"), nil
 }
+
+// LoadTagsFromFile loads tags from a file
+func LoadTagsFromFile(filePath string) ([]string, error) {
```

Comment:
At the moment I don't think it is possible to specify a list of tags? Only via the file? If so, possibly this is something we may want to add to the OTel config for the exporter?

Reply:
We could consider doing that, perhaps in a separate issue.

Reply:
+1

Reply:
https://github.com/jaegertracing/jaeger/issues/2301